### PR TITLE
Function.prototype.bind polyfill

### PR DESCRIPTION
**Pull request by dhorelik**
The problem is that PhantomJS v1.x does not support Function.prototype.bind. this results in js errors while running tests

### CODE REVIEW

File: cms/static/cms/js/polyfills/bind.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+//##############################################################################
```
Comment (FinalAngel) [inline cms/static/cms/js/polyfills/bind.js]: as this is specific to the test framework, can we add it to the testing folder structure?

**Comment by vxsx** [comment]
:+1:

File: cms/static/cms/js/polyfills/bind.js (hunk 1 above)
Comment (vxsx) [inline cms/static/cms/js/polyfills/bind.js]: even if it ends up in the actual cms bundle?

Comment (dhorelik) [inline cms/static/cms/js/polyfills/bind.js]: yep, but this will be added to the served bundle, my thought was to keep those in the same place

Comment (FinalAngel) [inline cms/static/cms/js/polyfills/bind.js]: I would not add it to our bundle, as it's actually a "hack" for phantomjs. What should ppl think when going through the core js code who find this?

Comment (dhorelik) [inline cms/static/cms/js/polyfills/bind.js]: I've tried injecting the polyfill in a different way, with the casperjs `clientScripts` or on `page.initialized` event. The problem is the code injected this way is executed too late, and the `.bind` throws errors anyway.

Comment (FinalAngel) [inline cms/static/cms/js/polyfills/bind.js]: I'm just saying to replace this file to the frontend/tests folder

**Pull request merged**